### ISSUE TITLE
autosync - provide clear feedback about pushes

### ISSUE THREAD

**Issue by dominikschulz** [justwatchcom/gopass#281]
The current git push/pull output is very _technical_. We should try to make it easier to grasp for less technical users.

Example:
```
gopass insert foo/bar
...
[git] Changes have been pushed to git@github.com:my/store.git
```

**Comment by jimb0bmij** [comment]
Is it possible to have a command like "gopass sync --store Foo" to make sure things are updated before we decide to make a change like "gopass edit/insert" or before you do gopass show do a auto pull just by doing "gopass"?

**Comment by dominikschulz** [comment]
You can always do `gopass git pull` but maybe this is too confusing.
Do you think it would provide a real benefit to alias `gopass git pull` to a command like `gopass sync`?

**Comment by jimb0bmij** [comment]
I think it would be a way to make it easier. Today I have to do "gopass git --store=Foo pull origin master". I need to type that each time or script it or history it. 

One reason for this is I'm trying to get my team to work with gopass. The road block we are hitting is people forget to "sync" before working with the data or someone state, I added x. The other person say I don't see that. Because they forgot to sync (IE pull the changes as they are just looking).

I offered up a cronjob idea to sync for them. Not perfect but better.
